app/src/test/java/org/schabi/newpipe/settings/ContentSettingsManagerTest.kt

```diff
@@ -0,0 +1,72 @@
+package org.schabi.newpipe.settings
+
+import android.content.SharedPreferences
+import org.junit.Assert
+import org.junit.Assume
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.junit.runners.Suite
+import org.mockito.Mockito
+import org.mockito.Mockito.`when`
+import org.mockito.junit.MockitoJUnitRunner
+import org.schabi.newpipe.settings.ContentSettingsManagerTest.ExportTest
+import java.io.File
+import java.io.ObjectInputStream
+import java.util.zip.ZipFile
+
+@RunWith(Suite::class)
+@Suite.SuiteClasses(ExportTest::class)
+class ContentSettingsManagerTest {
+
+    @RunWith(MockitoJUnitRunner::class)
+    class ExportTest {
+
+        private lateinit var preferences: SharedPreferences
+        private lateinit var newpipeDb: File
+        private lateinit var newpipeSettings: File
+
+        @Before
```

Comment:
Well from refactoring too much stuff around, i apparently created a `beforeClass` method annotated `@Before`. I'll fix this later on in my next PR.

Reply:
Please create a separate PR for this